tests/ttls

```diff
@@ -91,7 +91,7 @@ run_test() {
                 send \" NO EOF \n\";
                 exit 1;
             };
-        }"
+        }" || (wait_for_server_at_exit $SERVER_PID;  exit 1; )
 
     wait_for_server_at_exit $SERVER_PID
```

Comment:
Isn't this redundant now?

Reply:
I think this makes sure we wait for the server exit on success so I think it is still needed.

In case of the expect script fails, this line would not be executed and the script would terminate right away not waiting for the server and not giving the server logs.